### ISSUE TITLE
mesa fails to build because clang-libdir was removed

### ISSUE THREAD

**Issue by varris1** [chaotic-cx/nyx#62]
chaotic-nyx commit: 01e1cc0694bf547decbe9e742535096242f21b12
nixpkgs-unstable commit: 6806b63e824f84b0f0e60b6d660d4ae753de0477

`> meson.build:21:0: ERROR: Unknown options: "clang-libdir`

**Comment by PedroHLC** [comment]
Sorry, I've pushed the PR in a hurry because I was leaving for a trip. Never pushed without testing, the only time I did it broke 🤣

Won't be able to fix it before getting home. But one can easily roll it back, by adding this line to the flake's inputs:

```nix
{
  chaotic.inputs.mesa-git-src.url = "github:chaotic-cx/mesa-mirror/e805c9c05d39b45e63d30e670d517467b0dbec6b";
}
```

**Comment by PedroHLC** [comment]
Rolled back with https://github.com/chaotic-cx/nyx/commit/8abb2513ac2c405179997a80cff2a636beb93247.
We'll keep this open until a better solution appears.

**Comment by PedroHLC** [comment]
Properly fixed in 335d6406330f1379ef75d500b241c8924a9be80c
